bin/run-tests

```diff
@@ -28,7 +28,7 @@ execute_test () {
         echo "Running tests for ${exercise_name}";
 
         #remove the ignore line
-        sed -i 's/TEST_IGNORE();//' ./test_*.c
+        sed -i -e 's/TEST_IGNORE();//' ./test_*.c
```

Comment:
What's the reason for this change?

Reply:
I'm going to revert this and merge. If you think this change is necessary, please open another PR.

```suggestion
        sed -i 's/TEST_IGNORE();//' ./test_*.c
```

Reply:
Thanks for cleaning that up! I committed that accidentally, it's required for BSD's `sed` (I'm partially on macOS). But there are apparently more issues with that script that I wasn't able to fix yet. I've been meaning to do a separate PR with those fixes.